applications/projects/working-with-projects.adoc

```diff
@@ -8,6 +8,11 @@ toc::[]
 A _project_ allows a community of users to organize and manage their content in
 isolation from other communities.
 
+[NOTE]
+====
+Projects starting with `openshift-` and `kube-` are  xref:../../authentication/using-rbac.adoc#rbac-default-projects_using-rbac[default projects]. These projects host cluster components that run as pods and other infrastructure components. As such, {product-title} does not allow you to create Projects starting with `openshift-` or `kube-` using the `oc new-project` command. Cluster administrators can create these Projects using the xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#new-project[`oc adm new-project` command].
```

Comment:
s/pods/Pods/

Reply:
Use .adoc in xrefs.

Reply:
Thanks! Cut and paste error. I always do that, then wonder why Travis fails!